src/shared/components/common/paginator-cursor.tsx

```diff
@@ -0,0 +1,46 @@
+import { Component, linkEvent } from "inferno";
+import { I18NextService } from "../../services";
+import { PaginationCursor } from "lemmy-js-client";
+
+interface PaginatorCursorProps {
+  prevPage?: PaginationCursor;
+  nextPage?: PaginationCursor;
+  onNext(val: PaginationCursor): any;
+  onPrev(): any;
```

Comment:
We don't use what gets returned from this (if anything), so it could just return `void`.